### PR TITLE
10.0 migrate auth_supplier

### PR DESCRIPTION
**Pull request by StephanRozendaal**
Migrate the auth_supplier module to 10.0

### CODE REVIEW

File: auth_supplier/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {
     'name': "Auth Supplier",
     'category': 'Portal',
-    'version': '9.0.2.0.0',
+    'version': '10.0.1.0.0',
```
Comment (Fenkiou) [inline auth_supplier/__manifest__.py]: The module version was 2.0.0, it shouldn't be downgraded.

Comment (pedrobaeza) [inline auth_supplier/__manifest__.py:10]: No, it's correct as it is. The subversion should be reset on each new Odoo version.

Comment (Fenkiou) [reply auth_supplier/__manifest__.py:10]: This is non sense. Odoo releasing a version each year, modules won't have same version despite sharing exact same code.

Also, referring to [this](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers), there is no indication on resetting the module version.

The two first digits are just here to tell that the module can be installed only on a specific odoo version.

For this module, the version should even be `10.0.2.1.0` if you consider adding new translations is a feature.

**EDIT**: Forget my last sentence as it's transbot that produced the translation files and I guess the files are presents on version 9.0 too. Even if I think that new languages should bump version number too. :)

Comment (pedrobaeza) [reply auth_supplier/__manifest__.py:10]: Please see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#tasks-to-do-in-the-migration with the reference for the change of the version, and there's a rationale after this decision that is too large to explain here.

Comment (Fenkiou) [reply auth_supplier/__manifest__.py:10]: Oh, missed that one, sorry. I would like to read the discussion on how this decision came up if you have a link.

Comment (Fenkiou) [reply auth_supplier/__manifest__.py:10]: Just found the [issue](https://github.com/OCA/pylint-odoo/issues/59) talking about that, deleting my comment ;)

**EDIT**: Though deleting my comment would delete all comments but no :/
The first comment was:
> Module version was 2.0.0, it should not be downgraded

**EDIT2**: In fact, I'm not sure this issue is related to the decision taken

**Comment by Tardo** [comment]
Looks good for me... and I need it for migrate 'website_portal_purchase' to v10.0

**Review by Tardo — APPROVED** [review]
Looks good for me... and I need it for migrate 'website_portal_purchase' to v10.0

**Review by yajo — APPROVED** [review]
Code OK

**Review by felixvillafranca — COMMENTED** [review]
Now a field "Account Type" appears in the sign up form; but when sign up an error occurs:

screenshot 2017-05-30 at 17 47 13

**Comment by Tardo** [comment]
@felixvillafranca For some reason not show "Password" and "Confirm Password"... In local works fine.

signup

**Comment by pedrobaeza** [comment]
Check if there's any conflicting module

**Comment by felixvillafranca** [comment]
Hi Tardo; 
I've been checking your module; and you're rigth, the module is Ok..
Password and confirm password were hidden due to "auth_signup_verify_email" was installed.
And becuase of the outgoing mail server wasn't not configured it failed.

So, excuse me and thanks for your work.

**Review by felixvillafranca — APPROVED** [review]
It works

**Pull request merged**